### PR TITLE
Add Martha's martha_v3 endpoint to deploy, with note about allUsers WA-203

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

**Comment by kshakir** [comment]
See comments in WA-203 for more detail regarding `allUsers`.

**Comment by kshakir** [comment]
jenkins restart and hope this isn't the [definition of insantity](https://quoteinvestigator.com/2017/03/23/same/)

**Comment by kshakir** [comment]
jenkins retest and hope this isn't the [definition of insanity](https://quoteinvestigator.com/2017/03/23/same/)

**Comment by kshakir** [comment]
Created WA-209 with next steps

File: deploy.sh [hunk 1]
```diff
@@ -58,6 +58,21 @@ docker run --rm -v $PWD:${MARTHA_PATH} \
...
+#
+# For every new cloud functions deployed, across every environment, someone with owner access will need to run:
+#   ```
+#   gcloud beta functions \
+#     add-iam-policy-binding {FUNCTION_NAME} \
+#     --member=allUsers --role=roles/cloudfunctions.invoker \
+#     --project=broad-dsde-{ENVIRONMENT}
+#   ```
+# After an owner fixes the perms, the function contents may be overwritten by any editor, including the GCF-SA
+# `secret/dsde/martha/${ENVIRONMENT}/deploy-account.json`, and will stay public.
```
Comment (rsasch) [inline deploy.sh]: nit: I'm assuming GCF stands for Google Cloud Function service account, but it would be nice to spell it out

Comment (rsasch) [reply deploy.sh]: can you provide more details about how the owner would "[fix] the perms"?

**Review by salonishah11 — APPROVED** [review]

**Review by rsasch — APPROVED** [review]
LGTM 👍

**Comment by kshakir** [comment]
jenkins retest and hopefully find an agent with some disk space

**Pull request merged**